chatmessageview/src/main/kotlin/com/github/bassaer/chatmessageview/util/TimeUtils.kt

```diff
@@ -35,4 +35,15 @@ object TimeUtils {
         val millisOfDay = 1000 * 60 * 60 * 24
         return (timeDiff / millisOfDay).toInt()
     }
+
+    @JvmStatic
+    fun isSameDay(cal1: Calendar, cal2: Calendar): Boolean {
+        if (cal1 == null || cal2 == null) {
```

Comment:
I think this condition is always false, so this null check is unnecessary.

Reply:
Agree with you, just for the general utility code maturity. in case later on used by other case.

Reply:
okay. Thanks